common/src/app/common/pages/migrations.cljc

```diff
@@ -318,12 +318,12 @@
 (defmethod migrate 14
   [data]
   (letfn [(update-object [_ object]
-            (cond-> object
-              (and (not (= :text (:type object))) (nil? (:fills object)))
-              (set-fills)))
+                         (cond-> object
```

Comment:
why this indentation change?

Reply:
![image](https://user-images.githubusercontent.com/1579633/159881052-cf579929-534f-4a17-9233-59843b8d46ac.png)